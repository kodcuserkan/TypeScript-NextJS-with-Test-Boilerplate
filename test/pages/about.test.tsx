import React from 'react'
import { render } from '../testUtils'
import About from '../../pages/about'

describe('About page', () => {
  it('matches snapshot', () => {
    const { asFragment } = render(<About />, {})
    expect(asFragment()).toMatchSnapshot()
  })

  it('Checks whether h1 tag exists', () => {
    const { getByTestId } = render(<About />, {})
    expect(getByTestId('about-title')).toBeTruthy()
  })
})
