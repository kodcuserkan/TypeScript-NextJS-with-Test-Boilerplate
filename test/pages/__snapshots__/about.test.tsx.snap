// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About page matches snapshot 1`] = `
<DocumentFragment>
  <div>
    <h1
      data-testid="about-title"
    >
      About page
    </h1>
  </div>
</DocumentFragment>
`;
